src/io.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
```

Comment:
We chatted about this out of band, and @scalvert was going to file an issue to refactor to make the types happier around `options.shouldRemove!`

Reply:
Correct, will follow up with some fixes for the types.